src/socket.ts

```diff
@@ -19,21 +19,17 @@ export const connectCoinbase = () => {
       console.error(`Error encountered: ${err}`);
     });
 
-    connection.sendUTF(subscribeMessage);
+    connection.sendUTF(JSON.stringify(subscribeMessage));
 
     connection.on('close', () => {
       console.log(`Connection to ${process.env.COINBASE_URL} closed`);
     });
 
-    connection.on('message', message => {
+    connection.on('message', async message => {
       if (message.type === 'utf8' && message.utf8Data) {
-        const ethMessage = JSON.parse(message.utf8Data)
-        // the `sequence` property, which uniquely identifies a websocket message, is not sequential
-        // trade_id is sequential and every trade is sent
-        if ((ethMessage.trade_id)) {
-          const date = `${new Date().getHours()}:${new Date().getMinutes()}:${new Date().getSeconds()}`;
-          console.log(`[${date}]:`);
-          console.table(JSON.parse(message.utf8Data));
+        const parsedMessage = JSON.parse(message.utf8Data) as CreateTradeInput;

```

Comment:
Functional: Use natural type assignment instead of [type assertion](https://basarat.gitbook.io/typescript/type-system/type-assertion)

```suggestion
        const parsedMessage: CreateTradeInput = JSON.parse(message.utf8Data);
```